docs/environment_variables.md

```diff
@@ -79,3 +79,24 @@ SERVER_TRAFFIC_SECRET_0 XXXX
 CLIENT_HANDSHAKE_TRAFFIC_SECRET XXXX
 CLIENT_TRAFFIC_SECRET_0 XXXX
 ```
+
+`SSL_CERT_FILE` `REQUESTS_CA_BUNDLE` `CURL_CA_BUNDLE`
+-----------
+
+Valid values: a filename
+
+If one of these environment variables is set and `trust_env=True` and `verify=True` or `verify is None`, `verify` will be set by respectively `SSL_CERT_FILE`, `REQUESTS_CA_BUNDLE`, `CURL_CA_BUNDLE` environments as file.
```

Comment:
I reckon we ought to think about rephrasing this slightly. For one thing we should omit the `trust_env` note here.

Instead the opening paragraphs of the "Environment Variables" documentation should mention `trust_env`, noting that users can disable all variables here by setting `trust_env=False`